viewing_party/party.py

```diff
@@ -1,23 +1,176 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    if not (title and genre and rating):
+        return None
+    else:

```

Comment:
No need for this else statement. When possible we should try not to write extra statements or creating nesting when we don't need them to avoid bugs. 

Also, we can directly return a literal dictionary without referencing it with a variable, like: 
`return { "title" : title, "genre" : genre, "rating" : rating}`